package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@FunkyColdMedyna nice work setting up the boiler plate! 

No need to push up the package-lock.json in the repo, let's add this in our .gitignore